index.html

```diff
@@ -2245,7 +2245,7 @@ <h4 id="positioning_of_consecutive_opening_brackets_closing_brackets_comma_full_
+      <p its-locale-filter-list="en" lang="en"> In cases where multiple punctuation marks, such as <a class="characterClass" href="#cl-01">opening brackets (cl-01)</a>, <a class="characterClass" href="#cl-02">closing brackets (cl-02)</a>, <a class="characterClass" href="#cl-07">commas (cl-07)</a>, <a class="characterClass" href="#cl-06">full stops (cl-06)</a> and <a class="characterClass" href="#cl-05">middle dots (cl-05)</a>, come one after the other, the following space adjustments are made for aesthetic reasons (see [[[#fig2_13]]]). Note also that the half em and <a href="#term.quarter-em-space" class="termref2nd">quarter em spacing</a> added before or after punctuation marks, including the <a href="#term.half-em-space" class="termref2nd">half em spacing</a> after <a class="characterClass" href="#cl-06">full stops (cl-06)</a> appearing in the middle of a line, are subject, in principle, to line adjustment and may eventually be removed, except for those added after <a class="characterClass" href="#cl-06">full stops (cl-06)</a>. (See [[[#line_adjustment]]] for more about line adjustment.) For more information about the positioning of <a class="characterClass" href="#cl-02">closing brackets (cl-02)</a>, <a class="characterClass" href="#cl-06">full stops (cl-06)</a>, <a class="characterClass" href="#cl-07">commas (cl-07)</a> and <a class="characterClass" href="#cl-05">middle dots (cl-05)</a> at line end, see [[[#positioning_of_closing_brackets_full_stops_commas_and_middle_dots_at_line_end]]].</p>
```

Comment:
Do we need to change the term ("quarter em space") in the glossary as well? @kidayasuo @r12a @himorin

Reply:
it seems for me that we are better to agree on whether we will change all of 'space' to 'spacing' for mojikumi aki or not,, also fixes seems not to be consistent here for me...
original #85 mentions number of 'space(s)' are in doc and most of these seem to be, but number of them here around half(?).

Reply:
In reviewing this again, I agree on being consistent. Will standardize on "spacing" where possible as this is least ambiguous with inserting actual space characters.